daemon/daemon.go

```diff
@@ -231,6 +236,8 @@ func (c *Command) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			maintTransmitter.transmitMaintenance(errorKindSystemRestart, "system is restarting")
 		case state.RestartDaemon:
 			maintTransmitter.transmitMaintenance(errorKindDaemonRestart, "daemon is restarting")
+		case state.StopDaemon:
```

Comment:
And `state.RestartSocket`.. that may also be nice as it keeps the concept around "restart", per existing naming of functions and values.